components/AnimatedImage/library/src/commonMain/kotlin/org/jetbrains/compose/animatedimage/AnimatedImage.kt

```diff
@@ -2,13 +2,30 @@ package org.jetbrains.compose.animatedimage
 
 import androidx.compose.ui.graphics.ImageBitmap
 
+/**
+ * Represents an image formed by a sequence of frames with a specific duration each one.
+ */
 expect class AnimatedImage
 
+/**
+ * Loads an [AnimatedImage] from a given path.
+ */
 expect suspend fun loadAnimatedImage(path: String): AnimatedImage
 
+/**
+ * Loads an [AnimatedImage] from the resources.
+ */
 expect suspend fun loadResourceAnimatedImage(path: String): AnimatedImage
 
+/**
+ * Animates an [AnimatedImage] by returning an [ImageBitmap] for each frame of the image.
+ * The caller will be with the recomposed with each new frame that has been rendered.
```

Comment:
> The caller will be with the recomposed with each new frame that has been rendered.

Maybe "The caller will be recomposed with each new frame that has been rendered." is more correct.

Reply:
Oopsie, my sentence is completly incorrect. Not sure what I was thinking about! Will fix it in a moment

Reply:
Oh, you are right. But let's write that it is platform specific, and add info about each supported platform (we support only desktop at the moment)